spec/gutter-spec.coffee

```diff
@@ -0,0 +1,74 @@
+{Gutter, DEFAULT_PRIORITY} = require '../src/gutter'
```

Comment:
This is sorta wrong, but throughout this codebase as it currently stands we use camel case for constants in addition to constructors... so `DefaultPriority`. Maybe someday we should change them all.